### PR TITLE
fix: Add fucntion to call `trigger-workerflow` api.

### PR DESCRIPTION
**Pull request by rafiu007**

### CODE REVIEW

File: f8a_utils/ingestion_utils.py [hunk 1]
```diff
@@ -49,3 +54,17 @@ def unknown_package_flow(ecosystem: str, unknown_pkgs: Set[namedtuple]):
...
+
+
+def trigger_workerflow(payload):
+    """Triggering workerflow utility function."""
+    logger.debug("Triggered workerflow")
+
+    try:
+        _session.post(url=_WORKERFLOW_API_URL, json=payload)
+    except Exception as e:
+        logger.error('Failed to trigger unknown flow for payload %s with error %s',
```
Comment (jparsai) [inline f8a_utils/ingestion_utils.py:66]: correct the logger message. unknown flow error will indicate that something is wrong in ingestion API.

File: f8a_utils/ingestion_utils.py [hunk 2]
```diff
@@ -49,3 +54,17 @@ def unknown_package_flow(ecosystem: str, unknown_pkgs: Set[namedtuple]):
...
+                     payload, e)
+        raise Exception('Ingestion failed') from e
+    else:
+        logger.info('Ingestion call being executed')
```
Comment (jparsai) [inline f8a_utils/ingestion_utils.py:70]: same here

**Review by jparsai — CHANGES_REQUESTED** [review]

File: f8a_utils/ingestion_utils.py [hunk 3]
```diff
@@ -12,6 +12,11 @@
     host=os.environ.get("INGESTION_SERVICE_HOST", "bayesian-jobs"),
     port=os.environ.get("INGESTION_SERVICE_PORT", "34000"),
     endpoint='internal/ingestions/epv')
+
+_WORKERFLOW_API_URL = "http://{host}:{port}/{endpoint}".format(
+    host=os.environ.get("INGESTION_SERVICE_HOST", "bayesian-jobs"),
+    port=os.environ.get("INGESTION_SERVICE_PORT", "34000"),
+    endpoint='internal/ingestions/trigger-workerflow')
```
Comment (jparsai) [inline f8a_utils/ingestion_utils.py:19]: Keep the common URL is a variable and append //trigger-workerflow or /epv in function.

**Review by jparsai — CHANGES_REQUESTED** [review]

File: f8a_utils/ingestion_utils.py (hunk 3 above)
Comment (rafiu007) [reply f8a_utils/ingestion_utils.py]: done.

Comment (jparsai) [reply f8a_utils/ingestion_utils.py]: You could have given 'internal/ingestions/' also in URL as this is also common thing.

**Review by jparsai — APPROVED** [review]

**Pull request merged**